Gems/LyShine/Code/Editor/LyShineEditorSettings.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include "LyShineEditorSettings.h"
+
+#include <AzCore/Settings/SettingsRegistry.h>
+
+namespace LyShine
+{
+
+    namespace Internal
+    {
+        template<typename T>
+        void SetRegistry(const AZStd::string_view setting, T&& value)
+        {
+            if (auto* registry = AZ::SettingsRegistry::Get())
+            {
+                registry->Set(setting, AZStd::forward<T>(value));
+            }
+        }
+
+        template<typename T>
+        AZStd::remove_cvref_t<T> GetRegistry(const AZStd::string_view setting, T&& defaultValue)
+        {
+            AZStd::remove_cvref_t<T> value = AZStd::forward<T>(defaultValue);
+            if (const auto* registry = AZ::SettingsRegistry::Get())
+            {
+                T potentialValue;
+                if (registry->Get(potentialValue, setting))
+                {
+                    value = AZStd::move(potentialValue);
+                }
+            }
+
+            return value;
+        }
+    }
```

Comment:
update: Please don't duplicate these, just bring in the include from `ViewportSettings.h/cpp` (or move the functions to a separate file we can include in both places)

Reply:
umm I guess It should make sense to move this utility to SettingRegistry